app/addons/components/react-components.react.jsx

```diff
@@ -111,7 +112,7 @@ var ToggleHeaderButton = React.createClass({
   },
 
   render: function () {
-    var iconClasses = this.props.iconDefaultClass + ' ' + this.props.fonticon + ' ' + this.props.innerClasses,
+    var iconClasses = `${this.props.iconDefaultClass} ${this.props.fonticon} ${this.props.innerClasses}`,
```

Comment:
using `const` here would be cool. Also there are quite a few string concatenations in this function. I think we could use string templates to make them more readable. What do you think?